### PR TITLE
Update release processes to talk about prepare-release.sh

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

File: PROCESSES.md [hunk 1]
```diff
@@ -142,48 +143,56 @@ job.
...
-time.)
-
-5. Commit the changes. The first line of the commit message should
-be "Release {package_id} version {version}". Optionally, add release
-notes as the rest of the commit message, potentially by copying from
-the `history.md` file you've just edited. ([Sample
-commit](https://github.com/googleapis/google-cloud-dotnet/commit/3b580a6a0e8248daec4c84f6a45d0e07c094013d))
+accordingly. There is no automation for this step currently.
+
+5. Assuming you're only releasing a single place, run
```
Comment (amanda-tarafa) [inline PROCESSES.md]: nit: a single **package**

File: PROCESSES.md [hunk 2]
```diff
@@ -142,48 +143,56 @@ job.
...
+GitHub are the ones we want to release".
+
 **Create the release PR**
 
-1. Edit the API catalog (`apis/apis.json`). Find the package you
-want to release (by ID) and edit the version key to the new release
-version. Usually this will be a bump of prerelease (e.g.
-1.0.0-beta03 to 1.0.0-beta04) or a minor version bump (e.g. 1.3.0 to
-1.4.0). You may update any dependencies at the same time.
+1. Run `./prepare-release.sh set-version <api> <version>`, e.g.
```
Comment (amanda-tarafa) [inline PROCESSES.md]: nit: maybe instead of `<version>` -> `<new-version>`

File: PROCESSES.md [hunk 3]
```diff
@@ -142,48 +143,56 @@ job.
...
+`prepare-release.sh Google.Cloud.Speech.V1 2.0.0-beta01`. This
```
Comment (amanda-tarafa) [inline PROCESSES.md]: set-version is missing from this example.

File: PROCESSES.md [hunk 4]
```diff
@@ -142,48 +143,56 @@ job.
...
 Sample session when releasing Google.Cloud.Speech.V1:
 
 ```text
 $ git checkout -b release-speech
-$ pico apis/apis.json
-$ ./generateprojects.sh
-$ ./prepare-release.sh Google.Cloud.Speech.V1
-$ pico apis/Google.Cloud.Speech.V1/docs/history.md
-$ git commit -a
+$ ./prepare-release.sh set-version Google.Cloud.Speech.V1 2.0.0-beta03
```
Comment (amanda-tarafa) [inline PROCESSES.md]: super super nit: The example above is on Google.Cloud.Seech.V1 2.0.0-beta01 and this one is beta03 :)

File: PROCESSES.md (hunk 2 above)
Comment (jskeet) [reply PROCESSES.md]: Done.

File: PROCESSES.md (hunk 3 above)
Comment (jskeet) [reply PROCESSES.md]: Doh!

File: PROCESSES.md (hunk 1 above)
Comment (jskeet) [reply PROCESSES.md]: Done.

File: PROCESSES.md (hunk 4 above)
Comment (jskeet) [reply PROCESSES.md]: Thanks :)

**Pull request merged**